Project_Titanic/src/check/check_model.py

```diff
@@ -0,0 +1,39 @@
+from sklearn.metrics import accuracy_score
+from sklearn.metrics import precision_score
+from sklearn.metrics import recall_score
+from sklearn.metrics import confusion_matrix
+from sklearn.metrics import f1_score
+import os
+
+def check(model, X_test, Y_test, model_name):
```

Comment:
def check(model: find the type, X_test: pd.DataFrame, Y_test: pd.DataFrame, model_name: str) -> None: